packages/components/layout/src/Layout.styles.ts

```diff
@@ -1,6 +1,13 @@
 import tokens from '@contentful/f36-tokens';
 import { css } from 'emotion';
 import type { LayoutProps } from './Layout';
+import type { LayoutContextType } from './LayoutContext';
+
+export const NAVBAR_HEIGHT = 60;
```

Comment:
I prefer to avoid calculating sizes/positions based on external components we don't depend on in the package, but I understand we utilized all of our jokers' cards. If we go down that road, should we import this constant from the Navigation package to represent the relationship between the two?

Reply:
I agree with you, let me double check one last time if the usage of grid could solve this now. Otherwise I agree with you and we should set it up as a cross-package constant.